### ISSUE TITLE
Move to bookdown

### ISSUE THREAD

**Issue by Lextuga007** [nhs-r-community/statements-on-tools#12]
This was started in a README file but will quickly expand and possibly would be better in bookdown. I used the bookdown template to build my own https://github.com/Lextuga007/bookdown-template but the actions need updating and this would be better with the NHSR colour branding.

**Comment by bclarke-nes** [comment]
That's a good idea, particularly if we want to develop this to cover some of the other analytics challenges mentioned by @wbryant - happy to help, although I'm a beginner with bookdown. Question for now: I was planning to revise the current draft. Better to do that, then move to bookdown? Or move, then revise?

**Comment by Lextuga007** [comment]
Good question. Setting up bookdown will be fun but could take a bit of sorting out to get started with and the important thing is to revise the content so my suggestion is go for the revision in this basic format and then, if you want and have time, look to copying it over to bookdown. I've used the template I shared for an internal thing on Azure DevOps so it's pretty good, I'm just conscious it will have files in there that create broken GitHub Actions which are just annoying but not going to break the actual book.

**Comment by wbryant** [comment]
Yes I'm guessing bookdown could be set up independently of the README and other content pages, then just lifted and shifted once all the bookdown stuff is in place, so both could potentially be done at once?

**Comment by bclarke-nes** [comment]
Right, I'll have a stab at revising this afternoon.

**Comment by ChrisBeeley** [comment]
This issue was erroneously closed by #13

**Comment by tomjemmett** [comment]
@ChrisBeeley I can't remember if we had a conversation similar to this, but more along the lines of tying up a couple of related projects into one single bookdown/blogdown repo? In particular, this, and the [vision repo](https://github.com/nhs-r-community/nhs-r_vision), and perhaps the work I was going to get round to about install guides for commonly used analytical tools

**Comment by ChrisBeeley** [comment]
Yes, absolutely that

**Comment by cararthompson** [comment]
Hi folks, I'm going to give this a go. 

If you're looking to integrate different existing documents, I suggest we look into using child documents. 
https://bookdown.org/yihui/rmarkdown-cookbook/child-document.html
If that sounds like a good plan, I'll look into setting things up so that the bookdown script grabs the content from the readme / other files we point it to when we render the document, so that we don't end up duplicating content.

- Are all the other files publicly available?
- I found these, which I can use as a starting point, along with the README from this repo. Are there others at this point?
- Are there official brand guidelines you'd like me to apply in the bookdown script?

**Comment by wbryant** [comment]
Ooh, very exciting, child documents look like exactly what we want to do.  The other markdown document - [Open-Code.md](Open-code.md) - could be added.

From an organisational perspective it might make sense to move the bulk of the R/R package-specific material in the README to its own markdown, and use the README as the intro page to be viewed via Github directly.  Does that make sense @ChrisBeeley , @Lextuga007?

**Comment by cararthompson** [comment]
Can I check the desired output format? HTML doc?

**Comment by ChrisBeeley** [comment]
Yes, absolutely we would like a normal README, just "Hello, welcome to the repo, it's about *____*", contributing guidelines, whatever, and then separate chapters for the actual content.

And targeting HTML please, thanks. We can host with GitHub pages https://bookdown.org/yihui/bookdown/github.html.

Zoë and I can provide advice on this if necessary.

Thank you!

**Comment by cararthompson** [comment]
Perfect, thanks. I'll potter away at this and keep you posted on progress. The other easy hosting option is Netlify, but happy to GH pages if you prefer to use that. 

At the moment, I'm creating the bookdown doc inside this repo. Would it make more sense for it to be its own repo in due course, if it's going to pull in docs from a few different places?

**Comment by ChrisBeeley** [comment]
Haha! You're asking all the right questions!

Things are a little chaotic with respect to repos at the moment, I think for now let's just work here where the issues are and where we started, and I think eventually we will make a new super-repo that collates this and some other work.

Thanks 🙏

**Comment by cararthompson** [comment]
I've got the proof of concept working. Reading in the content of exisiting Rmd/md files was much more straightforward than I thought. I created a function that reads in any md/Rmd file we point it to, removes the title, and adds the rest of the content as a chapter. 

A few questions:
- Might be nice to have a Purpose chapter for the book?
- I think it would also be good to use GitHub Actions to render the book once a day so that it stays up to date with any changes in the other repos without needing to manually re-render it. 
- The vision files are all separate .md files. Do they exist anywhere as one document? If not, we can look into compiling them into one
- Do we want to replace "document" with "chapter" within each chapter? 
- Any strong feelings about whether to number sections/chapters?
- Anything else you'd like to see on the cover page (link to NHS-R twitter account?)

Anyway, pull request sent for the book as it stands. Happy to play with CSS at a later stage if we want to tweak its appearance.
